Vic2ToHoI4/Source/Hoi4Outputter/Map/OutBuilding.h

```diff
@@ -0,0 +1,19 @@
+#ifndef OUT_BUILDING_H
```

Comment:
in vic2 outputter I merged all of these standalone definitions into a single file. You could probably shave off 30 files with a single file with 30 lines.

Reply:
I plan to do another cleanup pass of the outputter when everything uses it. At that point I'll look into doing this.